spec/rspec/matchers/built_in/change_spec.rb

```diff
@@ -223,7 +223,7 @@ def ==(other)
   it "warns if passed a block using do/end instead of {}" do
     expect do
       expect {}.to change do; end
-    end.to raise_error(SyntaxError, /block passed to the `change` matcher/)
+    end.to raise_error(ArgumentError, /Block not received by the `change` matcher/)
```

Comment:
Well from our perspective it is unparsable! Also not being a `StandardError` doesn't bother us, we inherit from `Exception` for our own errors on purpose.